TCAT/AppDelegate.swift

```diff
@@ -151,7 +151,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
     }
     
-    // MARK: Helper Functions
+    // MARK: - Helper Functions
```

Comment:
Question: 
When is `MARK:` used usually?

Reply:
its usually used to denote a section in a file. when you add `// MARK: -` in a file, XCode will allow you to directly jump to that section quickly in the IDE.